src/pages/resultsView/mutualExclusivity/MutualExclusivityTable.spec.tsx

```diff
@@ -0,0 +1,59 @@
+import {assert} from 'chai';
+import React from 'react';
+import {formatPValue, formatPValueWithStyle, formatLogOddsRatio} from "./MutualExclusivityTable";
+
+describe("MutualExclusivityTable", () => {
+    describe("#formatPValue()", () => {
+        it("should return <0.001 for 0.0001", () => {
+            assert.equal(formatPValue(0.0001), "<0.001");
+        });
+
+        it("should return 0.001 for 0.001", () => {
+            assert.equal(formatPValue(0.001), "0.001");
+        });
+
+        it("should return 1.345 for 1.3454546", () => {
+            assert.equal(formatPValue(1.3454546), "1.345");
+        });
+
+        it("should return 0.050 for 0.05", () => {
+            assert.equal(formatPValue(0.05), "0.050");
+        });
+    });
+
+    describe("#formatPValueWithStyle()", () => {
+        it("should return <span>0.050</span> for 0.05", () => {
+            assert.deepEqual(formatPValueWithStyle(0.05), <span>0.050</span>);
+        });
+
+        it("should return <b><span>0.042</span></b> for 0.042", () => {
+            assert.deepEqual(formatPValueWithStyle(0.042), <b><span>0.042</span></b>);
+        });
+    });
+
+    describe("#formatLogOddsRatio()", () => {
+        it("should return <-3 for -6.32", () => {
+            assert.equal(formatLogOddsRatio(-6.32), "<-3");
+        });
+
+        it("should return -3.000 for -3", () => {
+            assert.equal(formatLogOddsRatio(-3), "-3.000");
+        });
+
+        it("should return 0.230 for 0.23", () => {
+            assert.equal(formatLogOddsRatio(0.23), "0.230");
+        });
+
+        it("should return 3.000 for 3", () => {
+            assert.equal(formatLogOddsRatio(3), "3.000");
+        });
+
+        it("should return >3 for 4.32", () => {
+            assert.equal(formatLogOddsRatio(4.32), ">3");
+        });
+
+        it("should return >3 for NaN", () => {
+            assert.equal(formatLogOddsRatio(NaN), ">3");
```

Comment:
`NaN` only happens when the sample list is empty, and it's not a valid case anyway. I did this because the old code was doing this, I'll just remove it because it's confusing.